src/server/main.ts

```diff
@@ -12,8 +12,7 @@ async function bootstrap() {
   const app = await NestFactory.create(AppModule, expressApp, {});
   setupClient(expressApp);
   await app.init();
-  await app.listen(port);
-  console.log(`server listening at http://localhost:${port}`);
+  await app.listen(port, () => { console.log(`server listening at http://localhost:${port}`); });
```

Comment:
Do we need to await and have a callback?

Reply:
await is not needed. The cb for log is not necessary but it will be better to put it there instead of another new line for logging purposes

Reply:
Oops, await is needed because `.listen` will start the app. `app.init` is not required/necessary to call.

Can we get rid of the express instance just for rendering the index.html?